### PR TITLE
fix helm install

### PR DESCRIPTION
**Pull request by zirain**
fix https://github.com/volcano-sh/volcano/issues/2217

```console
# helm install volcano  --namespace volcano-system helm/chart/volcano/
NAME: volcano
LAST DEPLOYED: Mon May  9 17:17:56 2022
NAMESPACE: volcano-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing volcano.

Your release is named volcano.

For more information on volcano, visit:
https://volcano.sh/
```

```console
# k get all -nvolcano-system
NAME                                TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
service/volcano-admission-service   ClusterIP   10.96.38.63    <none>        443/TCP    19s
service/volcano-scheduler-service   ClusterIP   10.96.119.36   <none>        8080/TCP   19s

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/volcano-admission     0/1     0            0           19s
deployment.apps/volcano-controllers   0/1     0            0           19s
deployment.apps/volcano-scheduler     0/1     0            0           19s

NAME                                            DESIRED   CURRENT   READY   AGE
replicaset.apps/volcano-admission-6c68cbbf98    1         0         0       19s
replicaset.apps/volcano-controllers-f4b69577b   1         0         0       19s
replicaset.apps/volcano-scheduler-c98cb745b     1         0         0       19s

NAME                               COMPLETIONS   DURATION   AGE
job.batch/volcano-admission-init   0/1                      19s
```

### CODE REVIEW

**Review by shinytang6 — APPROVED** [review]
/lgtm
/approve

**Review by shinytang6 — APPROVED** [review]
/lgtm

File: hack/generate-yaml.sh [hunk 1]
```diff
@@ -121,7 +121,7 @@ ${HELM_BIN_DIR}/helm template ${VK_ROOT}/installer/helm/chart/volcano --namespac
       >> ${DEPLOYMENT_FILE}
 
 ${HELM_BIN_DIR}/helm template ${VK_ROOT}/installer/helm/chart/volcano --namespace volcano-monitoring \
-      --name-template volcano --set basic.image_tag_version=${VOLCANO_IMAGE_TAG} \
+      --name-template volcano --set basic.image_tag_version=${VOLCANO_IMAGE_TAG} --set custom.metrics_enable=true \
```
Comment (shinytang6) [inline hack/generate-yaml.sh:124]: Maybe we should disable it by default?

Comment (zirain) [reply hack/generate-yaml.sh:124]: actually this will only render `volcanno-monitoring-latest.yaml`, which I think not in real use.

**Review by shinytang6 — APPROVED** [review]
/lgtm

**Pull request merged**